src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -99,6 +99,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       return;
     }
 
+    Boolean isDeleteConvo = Boolean.valueOf(request.getParameter("deleteConvo"));
```

Comment:
Rename to shouldDelete (no need for convo, it's obvious bc we are in conversation servlet)
Boolean -> boolean